test/build.spec.ts

```diff
@@ -115,7 +115,7 @@ describe('Project building', () => {
 				.to.have.property('layers')
 				.that.is.an('array')
 				.and.have.length(1);
-
+			// tslint:disable-next-line:no-unused-expression
 			expect(image).to.have.property('error').that.is.not.null;
```

Comment:
`resin-lint` bump to `3.0.1`: no-unused-expression: unused expression, expected an assignment or function call